README.md

```diff
@@ -36,27 +34,26 @@ Constraints to describe the map coloring problem:
 * Each region can only select one color
 * No regions sharing a border can share a color
 
-Code Specifics
---------------
-Why only four colors?
-~~~~~~~~~~~~~~~~~~~~~
+## Code Specifics
+
+### Why only four colors?
+
 * In the code, we let each of the regions choose one among four colors. Why
-  are we limiting ourselves to only four colors? According to the `Four Colour
-  Theorem <https://en.wikipedia.org/wiki/Four_color_theorem>`_, we need no more
+  are we limiting ourselves to only four colors? According to the [Four Colour
+  Theorem](https://en.wikipedia.org/wiki/Four_color_theorem), we need no more
   than four colors to color any planar map such that no two adjacent regions
   share the same color
 
-Alternative implementation
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+### Alternative implementation
+
 * To solve the map coloring problem, in this demo, we are forming constraints,
   converting said constraints into a BQM, and then feeding said BQM to a solver.
   Please note, however, that the D-Wave Ocean stack also has a function called
-  `min_vertex_color_qubo(..) <https://docs.ocean.dwavesys.com/projects/
-  dwave-networkx/en/latest/reference/algorithms/generated/dwave_networkx
-  .algorithms.coloring.min_vertex_color_qubo.html#dwave_networkx.algorithms
-  .coloring.min_vertex_color_qubo>`_ that will directly form the map coloring
-  BQM for you.
-
-License
--------
-Released under the Apache License 2.0. See `LICENSE <./LICENSE>`_ file.
+  [min_vertex_color_qubo(..)][1] that will directly form the map coloring BQM
+  for you.
+
+## License
+
+Released under the Apache License 2.0. See [LICENSE](./LICENSE) file.
+
+[1]: https://docs.ocean.dwavesys.com/projects/dwave-networkx/en/latest/reference/algorithms/generated/dwave_networkx.algorithms.coloring.min_vertex_color_qubo.html#dwave_networkx.algorithms.coloring.min_vertex_color_qubo
```

Comment:
I couldn't figure out how to wrap the long link without breaking it, so I just added a reference-style link for readability.